config/locales/en.yml

```diff
@@ -645,16 +645,21 @@ en:
       email_body: |
         Hello %{preferred_name},
 
-        Thanks for starting your taxes with GetYourRefund - you’re almost there! We will prepare your taxes once we have all of your required information and tax documents.
+        Thanks for starting your taxes with GetYourRefund - you’re almost there! Your case number is %{client_id}. We will prepare your taxes once we have all of your required information and tax documents.
 
-        If you have more documents to send or have any questions about your case you can email your tax preparer at hello@getyourrefund.org, chat with our specialists using the chat box on getyourrefund.org/tax-questions or submit more documents at your secure submission link: %{requested_docs_link}
+        You can securely upload your ID and additional tax documents here: %{requested_docs_link}
+
+        If you have any questions you can contact your tax team via email <a href="mailto:hello@getyourrefund.org>hello@getyourrefund.org</a> or text message 58750.
```

Comment:
The footer links to support@getyourrefund.org . Are you sure this is the right email address? If so, great, but I wanted to check.

Reply:
Yes - seems that this link goes directly to their tax preparer so we want to use hello@